uk/guide/migrating-4.md

```diff
@@ -419,7 +419,7 @@ Running the above `npm` command will update `package.json` as follows:
     "errorhandler": "^1.1.1",
     "express": "^4.8.0",
     "express-session": "^1.7.2",
-    "jade": "^1.5.0",
+    "pug": "^1.5.0",
```

Comment:
It's okay, what i was asking is, what should i do about the jade instances, should i just go back and leave jade there, or should i leave it as is, because pug don't have older versions. Maybe i should just go and put `pug` & input it's current version.